src/analytics/message_info.rs

```diff
@@ -0,0 +1,14 @@
+use {parquet_derive::ParquetRecordWriter, serde::Serialize, std::sync::Arc};
+
+#[derive(Debug, Clone, Serialize, ParquetRecordWriter)]
+#[serde(rename_all = "camelCase")]
+pub struct MessageInfo {
```

Comment:
@biranyucel, here is all the stuff we are exporting for analytics. Look good? Also anything else you could think you want?

Reply:
Don't we have the time information?

Reply:
We can get the time info! Do you want time recieved?

Reply:
Okay, with the client-wallet pair, I think most of that information could be gathered from relay_messages on our side if we need it.    
Based on our chat, your suggestion to have a client registration date was so good. Can we add that, please?